webapp/alembic/versions/2ab5564cfe99_add_assets.py

```diff
@@ -0,0 +1,35 @@
+"""add_assets
+
+Revision ID: 2ab5564cfe99
+Revises: eb9f8639d610
+Create Date: 2020-01-09 14:37:08.316607
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "2ab5564cfe99"
+down_revision = "eb9f8639d610"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
If you've decided this is the correct version of this fixture, then we should probably remove the "please adjust!" comment.

Reply:
Removed